templates/index.twig

```diff
@@ -6,10 +6,18 @@
 
 {% extends '_layouts/default' %}
 
+{% block styles %}
+    {% include '_shared/styles' %}
+{% endblock %}
+
 {% block page %}
     {% include [
         '_entries/' ~ entry.section.handle ~ '/' ~ entry.type,
         '_entries/' ~ entry.section.handle ~ '/default',
         '_entries/default'
     ] %}
 {% endblock %}
+
+{% block scripts %}
+    {% include '_shared/scripts' %}
```

Comment:
Why here rather than in `_layouts/default`

Reply:
My reasoning was that the **default layout** defines blocks where css and scripts should be placed in the page. The same way it has a page block to define where the page content should be placed.

The **index.twig** which extends the **default layout** has its own logic to handle that.

So we could technically have another template file that also extends the **default layout** which could have its own logic to define extra javascript/css.

Reply:
Fair enough. But the default layout will no longer include the WET-BOEW. Every template extending the default layout will require the css/js logic.

Reply:
That's a good point. It's a bit of a trade off I guess. The default layout is now more flexible in the sens that we could have custom pages that do not use the WET-BOEW library.

Do you want me to move them back in the default layout?